UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/IAzureWebJobHealthCheckService.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    public interface IAzureWebJobHealthCheckService
+    {
+        public Task<HealthCheckResult> CheckHealthAsync(CancellationToken cancellationToken = default);
+    }
```

Comment:
access modifiers not required in interface for methods.

Reply:
Removed